source/utils/LogManager.h

```diff
@@ -18,45 +18,55 @@
 #ifndef LOGMANAGER_H
 #define LOGMANAGER_H
 
+#include <memory>
```

Comment:
Are theses needed ? Moreover, in our coding guide, standard headers should be included at last. The order should be:
1 - Class header (for cpp files)
2 - OD headers
3 - Library headers (ogre, cegui, sfml, ...)
4 - standard headers

This do not prevent merge but if you want to go for something else, it's good to know ^^